hubmap_commons/type_client.py

```diff
@@ -110,20 +140,29 @@ def __init__(self, info: Dict[str, Any]):
         The dict must match the format produced by self.to_json().
         """
         # Needs to set self.name, self.description, self.primary
+        #pprint(info)
         self.name = info['name']
         self.description = info['description']
         self.primary = info['primary']
         self.vitessce_hints = (info['vitessce-hints']
                                if 'vitessce-hints' in info
                                else [])
+        self.contains_pii = (info['contains-pii']
+                             if 'contains-pii' in info
+                             else True)  # Fail to True for safety
+        self.vis_only = (info['vis-only']
+                         if 'vis-only' in info
+                         else False)  # False is more common
```

Comment:
The logic from [search-api](https://github.com/hubmapconsortium/search-api/pull/326/files#diff-08b762a9253c661ed18ef9f5be5b1c7ae544587bd030b3b4fd6e84f9ebfeed60R127-R128) is being duplicated. I think it should only live in one place.

Reply:
That part is intentional, since the defaults are meant to hold unless specifically overridden by information from the version in search-api.  Enforcing the defaults here cases 'safe' behavior even if the version of search-api which is queried does not yet support the additional attributes. (Your comment that I should be using .get() from the search-api PR should apply here however.